src/form/form-password-input.tsx

```diff
@@ -0,0 +1,27 @@
+import { useField } from 'formik';
+import React, { FC } from 'react';
+
+import { StyledPasswordInput } from '../components/styled-password-input/styled-password-input';
+import { ErrorMessage } from './error-message/error-message';
+
+interface Props {
+  name: string;
+}
+
+export const FormPasswordInput: FC<Props> = ({ name }) => {
+  const [field, meta] = useField<string>(name);
+  const hasError = meta.touched && meta.error !== undefined;
```

Comment:
create `hasError` util, and reuse it in another `Form<Components>`.  
Change naming for  `isError={hasError}` (single name) and pass it inside  `ErrorMessage` too.